.travis.yml

```diff
@@ -4,9 +4,10 @@
 
 language: node_js
 node_js:
+  - 'node'
```

Comment:
What does this mean? Which version is it going to be?

Reply:
`node` is the current stable which is `11` at the moment,

Reply:
`node` is the current stable which is `11` at the moment.

Reply:
Then please remove `11`?